src/main/java/me/ryanhamshire/GriefPrevention/FlatFileDataStore.java

```diff
@@ -320,6 +320,11 @@ else if (this.getSchemaVersion() == 0)
                         List<String> containerNames = Arrays.asList(line.split(";"));
                         containerNames = this.convertNameListToUUIDList(containerNames);
 
+                        //fifth line is list of players who can access creatures
+                        line = inStream.readLine();
```

Comment:
This is the sixth line (not fifth, as the copied comment indicates), which is already used for access trust data. This unexpectedly elevates permissions for users.